### ISSUE TITLE
Rendering failed when exporting notebook to html

### ISSUE THREAD

**Issue by ricgu8086** [plotly/plotly_express#58]
I just did a toy example with Iris dataset and parallel_coordinates function (exactly as in the documentation) in a jupyter notebook and it worked. But when I exported the notebook as html, instead of the plot there was just an empty white space.

**Comment by nicolaskruchten** [comment]
That's odd... the documentation itself is generated by exporting the notebook to HTML :)

Some questions:

1. What versions of Python, Jupyter and Plotly Express are you using?
2. How are you exporting to HTML? Via the command line with `nbconvert` or some other way
3. When you open up the resulting HTML file, are there any javascript errors in the browser console?

**Comment by ricgu8086** [comment]
Solved :-)

While checking versions and test if "nbconvert" command produces the same result as the "Download as.." menu, I executed everything again, clicked on "Save an checkpoint" and exported it using nbconvert, and it worked this time. The "bug" was that I didn't properly saved the file (usually this is automatic), and the confusion came because the whole code along with the results was in the html, but not the plot, just the space where the plot should be located. Probably a bad timing issue related with autosaving, maybe a cache error.
